modules/debts_api/app/controllers/debts_api/v0/financial_status_reports_controller.rb

```diff
@@ -10,6 +10,8 @@ class FinancialStatusReportsController < ApplicationController
       service_tag 'financial-report'
       before_action { authorize :debt, :access? }
 
+      STATSD_KEY_PREFIX = 'api.dmc'
```

Comment:
Only issue I see is here with the STATSD_KEY_PREFIX.

I see an example from the 1010EZ here:
app/controllers/v0/form1010_ezrs_controller.rb:24

Maybe we could do something like that and call it something like: `api.5655`